tests/unit/test.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import AddMemorialForm from '@/components/AddMemorialForm.vue'
+import VueI18n from 'vue-i18n'
+
+describe("addmemorial testl", () => {
+    let localVue = createLocalVue()
+    localVue.use(VueI18n)
+    test = {
+      name: 'kevin smith',
+      gender: 'male',
+      photo_upload: 'no',
+      photo_base64: 'nan',
+      birth_date: '25-11-1954',
+      passing_date: '25-11-2020',
+      location: 'jaipur::rajasthan',
+      prompt: 'none',
+      prompt_response: 'none'
+  }
+    const i18n = new VueI18n({
+      locale: 'en',
+      fallbackLocale: 'en'
+      
```

Comment:
stray space

Reply:
yeah I am extremely sorry about all these formatting etc. but this was just a demo test for you to look. i need to clean the code and rewrite it so it is acceptable for merging. thanks